cmd/repl/repl.go

```diff
@@ -81,9 +83,25 @@ func render(rsp interface{}, table *tablewriter.Table) bool {
 	return isTable
 }
 
+func flagPassed(name ...string) bool {
+	found := false
+	for _, n := range name {
+		flag.Visit(func(f *flag.Flag) {
+			if f.Name == n {
+				found = true
+			}
+		})
+	}
+	return found
+}
+
 func main() {
 	ds := flag.String("datasource", "", "database connect string")
 	modelDir := flag.String("model_dir", "", "model would be saved on the local dir, otherwise upload to the table.")
+	cliStmt := flag.String("execute", "", "execute SQLFlow from command line.  e.g. --execute 'select * from table1'")
```

Comment:
FYI, Go's default "flag" package doesn't differentiate between `-` and `--`, so `-execute` is the same as `--execute`.